src/middlewares/query.check.js

```diff
@@ -28,8 +28,8 @@ export const searchArticleQuerybuilder = (searchQueries) => {
 };
 
 export const checkQuery = (req, res, next) => {
-  const {
```

Comment:
Why is this code removed?

Reply:
> Why is this code removed?

yes, I removed it because the offset is stated from `limit and page queries`  not from queries. @nkalyesubula  hope that I am clear to the point.